test/test_suites.py

```diff
@@ -81,206 +129,492 @@ def run_in_confluent(self):
-        )),
-        ("TestKcDeleteCreate", EndToEndTestSuite(
-            test_instance=TestKcDeleteCreate(driver, nameSalt), run_in_confluent=True, run_in_apache=True
-        )),
-        ("TestKcDeleteCreateChaos", EndToEndTestSuite(
-            test_instance=TestKcDeleteCreateChaos(driver, nameSalt), run_in_confluent=True,
-            run_in_apache=True
-        )),
-        ("TestKcDeleteResume", EndToEndTestSuite(
-            test_instance=TestKcDeleteResume(driver, nameSalt), run_in_confluent=True, run_in_apache=True
-        )),
-        ("TestKcDeleteResumeChaos", EndToEndTestSuite(
-            test_instance=TestKcDeleteResumeChaos(driver, nameSalt), run_in_confluent=True,
-            run_in_apache=True
-        )),
-        ("TestKcPauseCreate", EndToEndTestSuite(
-            test_instance=TestKcPauseCreate(driver, nameSalt), run_in_confluent=True, run_in_apache=True
-        )),
-        ("TestKcPauseCreateChaos", EndToEndTestSuite(
-            test_instance=TestKcPauseCreateChaos(driver, nameSalt), run_in_confluent=True,
-            run_in_apache=True
-        )),
-        ("TestKcPauseResume", EndToEndTestSuite(
-            test_instance=TestKcPauseResume(driver, nameSalt), run_in_confluent=True, run_in_apache=True
-        )),
-        ("TestKcPauseResumeChaos", EndToEndTestSuite(
-            test_instance=TestKcPauseResumeChaos(driver, nameSalt), run_in_confluent=True,
-            run_in_apache=True
-        )),
-        ("TestKcRecreate", EndToEndTestSuite(
-            test_instance=TestKcRecreate(driver, nameSalt), run_in_confluent=True, run_in_apache=True
-        )),
-        ("TestKcRecreateChaos", EndToEndTestSuite(
-            test_instance=TestKcRecreateChaos(driver, nameSalt), run_in_confluent=True, run_in_apache=True
-        )),
-        ("TestKcRestart", EndToEndTestSuite(
-            test_instance=TestKcRestart(driver, nameSalt), run_in_confluent=True, run_in_apache=True
-        )),
-        ("TestSchemaEvolutionDropTable", EndToEndTestSuite(
-            test_instance=TestSchemaEvolutionDropTable(driver, nameSalt), run_in_confluent=True,
-            run_in_apache=True
-        )),
-        ("TestSchemaEvolutionMultiTopicDropTable", EndToEndTestSuite(
-            test_instance=TestSchemaEvolutionMultiTopicDropTable(driver, nameSalt), run_in_confluent=True,
-            run_in_apache=True
-        )),
-        ("TestStreamingClientParameterOverride", EndToEndTestSuite(
-            test_instance=TestStreamingClientParameterOverride(driver, nameSalt), run_in_confluent=True,
-            run_in_apache=True
-        )),
-    ])
-
-    # Return all suites or only selected subset
-    if allowedTestsCsv is None or allowedTestsCsv == "":
-        return test_suites
-    else:
-        testsToRun = dict((k, v) for k, v in test_suites.items() if k in allowedTestsCsv.split(','))
-        print("Running", len(testsToRun), "tests")
-        return testsToRun
+    """
+    return OrderedDict(
```

Comment:
That's only adding `cloud_platform=CloudPlatform.ALL,` to all suites and code formatter.